neural_compressor/torch/algorithms/weight_only/autoround.py

```diff
@@ -12,27 +12,156 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import copy
+import json
 import time
+from typing import Union
 
 import torch
 from auto_round import AutoRound  # pylint: disable=E0401
-from auto_round.calib_dataset import CALIB_DATASETS  # pylint: disable=E0401
-from auto_round.utils import get_block_names  # pylint: disable=E0401
+from auto_round.calib_dataset import get_dataloader  # pylint: disable=E0401
+from auto_round.export.export_to_itrex.model_wrapper import WeightOnlyLinear  # pylint: disable=E0401
+from auto_round.utils import get_module, quant_weight_w_scale, set_module  # pylint: disable=E0401
 
 from neural_compressor.torch.algorithms import Quantizer
-from neural_compressor.torch.utils import logger
+from neural_compressor.torch.utils import get_accelerator, is_transformers_imported, logger
+
+if is_transformers_imported():
+    import transformers
+
+
+def pack_model(
```

Comment:
May I know the difference between this `pack_model` and `pack_model` from auto-round? Can we reuse `pack_model` from auto-round?

Reply:
now import from AutoRound. https://github.com/intel/auto-round/pull/118